airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -126,16 +126,27 @@ def backoff_time(self, response: requests.Response) -> Optional[float]:
         # we again could have 5000 per another hour.
 
         min_backoff_time = 60.0
-
         retry_after = response.headers.get("Retry-After")
+        # TODO: refactor
```

Comment:
have we refactored it already or it's going to be made in the future? if so, provide the details next to this TODO

Reply:
updated